plugins/gcloud/lib/samson_gcloud/image_scanner.rb

```diff
@@ -28,14 +29,11 @@ def result_url(image)
 
       def status(id)
         case id
-        when WAITING
-          "Waiting for Vulnerability scan"
-        when SUCCESS
-          "No vulnerabilities found"
-        when FOUND
-          "Vulnerabilities found"
-        when ERROR
-          "Error retrieving vulnerabilities"
+        when WAITING then "Waiting for Vulnerability scan"
+        when SUCCESS then "No vulnerabilities found"
+        when FOUND then "Vulnerabilities found"
+        when ERROR then "Error retrieving vulnerabilities"
+        when UNSUPPORTED then "Only full gcr repos with shas are supported for scanning"
         else raise
```

Comment:
can we display failing id in the exception?

Reply:
The message is shown per build, so it will be clear which build failed

Reply:
ID is a phase in this case. And raise is to catch unsupported phase.